pkg/controllers/dynakube/extension/secret.go

```diff
@@ -0,0 +1,83 @@
+package extension
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/dttoken"
+	k8ssecret "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/secret"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	eecTokenKey         = "eec-token"
+	eecTokenValuePrefix = "EEC dt0x01"
+	secretSuffix        = "-extensions-token"
+)
+
+func reconcileSecret(ctx context.Context, dk *dynakube.DynaKube, kubeClient client.Client, apiReader client.Reader) error {
+	log.Info("reconciling secret")
```

Comment:
Nitpick, but what do you think about being a little bit more specific here?

Reply:
Because the logger already has the `extension` value, so we already know it's reconciling the secret from extensions.

But I will update the comment!